vividus-plugin-applitools/src/main/resources/properties/defaults/default.properties

```diff
@@ -1,6 +1,7 @@
 applitools.server-uri=
 applitools.execute-api-key=
 applitools.app-name=Application under test
+applitools.execution-cloud.batch-name=VIVIDUS test run
```

Comment:
why is it needed?

Reply:
<img width="841" alt="image" src="https://github.com/vividus-framework/vividus/assets/30923336/644935c2-4e51-47b7-820c-4e5f5c09a263">

to group stories under one batch